nucypher/cli/literature.py

```diff
@@ -125,8 +126,25 @@
 #
 # Events
 #
+
 CONFIRM_OVERWRITE_EVENTS_CSV_FILE = "\nOverwrite existing CSV file - {csv_file}?"
 
+#
+# Remove Unused
+#
+
+
+FETCHING_INACTIVE_STAKES = 'Fetching inactive stakes'
+
+NO_INACTIVE_STAKES = "No inactive stakes found\n"
+
+CONFIRM_REMOVE_ALL_UNUSED_SUBSTAKES = """
+This action will perform a series of transactions removing all unused substakes
+(Indices {stakes}).
+
+Confirm removal of all {quantity} unused substakes?"""
```

Comment:
```suggestion
CONFIRM_REMOVE_ALL_UNUSED_SUBSTAKES = """
This action will perform a series of transactions to remove all unused sub-stakes
(Indices {stakes}).

Confirm removal of {quantity} unused sub-stakes?"""
```

Reply:
Updated.  Let me know what you think of the new version.